net-analyzer/chronograf/chronograf-1.8.5-r1.ebuild

```
@@ -14,12 +16,12 @@ SRC_URI="https://github.com/influxdata/chronograf/archive/${PV}.tar.gz -> ${P}.t
 LICENSE="AGPL-3+"
 SLOT="0"
 KEYWORDS="~amd64"
-IUSE=""
 
-COMMON_DEPEND="acct-group/chronograf
-	acct-user/chronograf"
-DEPEND="${COMMON_DEPEND}"
-RDEPEND="${COMMON_DEPEND}"
+DEPEND="
+	acct-group/chronograf
+	acct-user/chronograf
+"
+RDEPEND="${DEPEND}"
```

You should honor the maintainer's style here (since this package does have a maintainer),
```suggestion
DEPEND="acct-group/chronograf
	acct-user/chronograf"
RDEPEND="${DEPEND}"
```